examples/reference/integration/tests/test_upgrade.py

```diff
@@ -0,0 +1,89 @@
+import dcos.http
+import pytest
+import re
+import shakedown
+
+from tests.test_utils import (
+    PACKAGE_NAME,
+    check_health,
+    install,
+    marathon_api_url_with_param,
+    request,
+    run_dcos_cli_cmd,
+    spin,
+    uninstall,
+)
+
+
+MASTER_CUSTOM_NAME='Master Custom'
+# TODO: replace this URL once we have a stock version from which to upgrade
+MASTER_CUSTOM_URL='https://infinity-artifacts.s3.amazonaws.com/reference-latest/stub-universe-reference.zip'
+
+
+def setup_module(module):
+    uninstall()
+
+
+def teardown_module(module):
+    uninstall()
+
+
+@pytest.mark.upgrade
+def test_upgrade():
+    test_version = get_pkg_version()
+    print('Found test version: {}'.format(test_version))
+    add_repo(test_version)
+    master_version = get_pkg_version()
+    print('Found master version: {}'.format(master_version))
+    print('Installing master version')
+    install(master_version)
+    check_health()
+    print('Upgrading to test version')
```

Comment:
shouldn't remove_repo(master_version) be done here? otherwise it looks like it'd just reinstall the test version